libandroid-navigation/src/test/java/com/mapbox/services/android/navigation/v5/navigation/ElapsedTimeTest.java

```diff
@@ -8,19 +8,25 @@
 
 public class ElapsedTimeTest {
 
+  private static final double DELTA = 1E-10;
+
   @Test(expected = NavigationException.class)
   public void errorThrownIfEndCalledBeforeStart() {
     new ElapsedTime().end();
   }
 
   @Test
-  public void elapsedTime() {
+  public void elapsedTime_returnsCorrectTimeInSeconds() {
     ElapsedTime elapsedTime = new ElapsedTime();
+
     elapsedTime.start();
     elapsedTime.end();
```

Comment:
It'd be great to add an "empty" method making explicit what's going on here i.e. I think it'd make clear that an operation happens in between `start` and `end` and we're calculating the diff.

Reply:
I think this is OK as-is

Reply:
Yeah, you're right I overlooked it because there's a bunch of "pre" assert related code. Sorry for the confusion 😬

Reply:
Sorry above comment was referring to a different convo 👀 https://github.com/mapbox/mapbox-navigation-android/pull/1731/files#r256133569